include/wil/com_apartment_variable.h

```diff
@@ -212,7 +212,7 @@ namespace wil
             template<typename T>
             std::any& get_or_create(any_maker<T>&& creator)
             {
-                apartment_variable_storage* variable_storage{};
+                apartment_variable_storage* variable_storage;

```

Comment:
Should revert these. Let the compiler optimize the unnecessary initialization away.

Reply:
This is a pointer though. Not needed.

Reply:
Yes, it's needed. Pointers don't default-init to nullptr.